pkg/controllers/clusterresourceplacementeviction/controller.go

```diff
@@ -271,7 +271,12 @@ func isEvictionAllowed(bindings []placementv1beta1.ClusterResourceBinding, crp p
 	}
 
 	var desiredBindings int
-	placementType := crp.Spec.Policy.PlacementType
+	var placementType placementv1beta1.PlacementType
+	if crp.Spec.Policy != nil {
```

Comment:
Hi Arvind! Just a nit: we could use PickAll as a default and set the value if policy is not nil (to avoid empty strings being set a default).